### DISCUSSION TITLE
Datahub Export - HTTP Delivery with PUT

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17653] [Q&A / Support] [answered]
I am looking to use Datahub's HTTP Delivery against a `PUT` endpoint. Is it possible to build the URL dynamically, using a field on the object class? For example `PUT` against: `https://my-url.app/products/<product_id>`? 

While `POST` might not need the `product_id`, a `PUT` will be against the resource itself.

**Comment by fashxp** [comment, marked as answer]
No, this is not possible out of the box yet.

**Reply by cancan101** [reply]
Should I file a feature request ticket for this?

**Reply by fashxp** [reply]
yes please. please also share some more context, use cases and examples. thanks.

**Reply by cancan101** [reply]
Posted: #17757
